JSON/The Modkit.json

```diff
@@ -13,5 +14,6 @@
   "Souvenir": { "Status": "NotACandidate" },
   "SteamID": "2137793443",
   "Symbol": "Mdk",
+  "TutorialVideoUrl": "https://youtu.be/-hHdQ62V33k",
```

Comment:
Tutorial videos should not use the youtu.be domain, and should instead use youtube.com, for consistency.

Reply:
- red031000, 2019